### PR TITLE
Fixes #7244: Change request list page, link to Rules are invalids (extra rudder)

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/7244

### CODE REVIEW

File: rudder-web/src/main/scala/com/normation/rudder/web/model/JsInitContextLinkUtil.scala [hunk 1]
```diff
@@ -130,7 +130,7 @@ object JsInitContextLinkUtil extends Loggable {
 
   def createGroupLink(id:NodeGroupId) = {
     roGroupRepository.getNodeGroup(id) match {
-      case Full((group,_)) => <span> {SHtml.a(() => S.redirectTo(groupLink(id)), Text(group.name))} (Rudder ID: {id.value})</span>
+      case Full((group,_)) => <span> <a href={baseGroupLink(id)}>{group.name}</a> (Rudder ID: {id.value})</span>
```
Comment (ncharles) [inline rudder-web/src/main/scala/com/normation/rudder/web/model/JsInitContextLinkUtil.scala]: did you check all the calls made from this function ? i'm pretty sure only change requests are broken, but others part call createRuleLink (like, the diff displayer)

**Comment by VinceMacBuche** [comment]
PR updated

File: rudder-web/src/main/scala/com/normation/rudder/web/model/JsInitContextLinkUtil.scala (hunk 1 above)
Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/model/JsInitContextLinkUtil.scala]: I verified the other calls are ok!

I still don't know why it adding /rudder ...)

**Comment by fanf** [comment]
I'm really not at ease to to know why the context is added in that case. Still, the correction work, and I don't know where to look anywhere else. 
So let's merge it.

**Pull request merged**